preprocessy/outliers/handleoutlier.py

```diff
@@ -0,0 +1,100 @@
+class HandleOutlier:
+
+
+    """ Class for handling outliers on its own or according to users needs.
+
+        Private methods
+       _ _ _ _ _ _ _ _ _ _
+
+       __returnQuartiles() : returns the 5% and 95% mark in the distrubution 
```

Comment:
No such method exists in this class. The method name is `__return_quartiles`.

Reply:
Will correct